packages/app/src/cli/services/dev/extension/bundler.ts

```diff
@@ -20,6 +20,11 @@ export interface FileWatcherOptions {
   payloadStore: ExtensionsPayloadStore
 }
 
+interface Overrides {
+  contents?: string
+  outputPath?: string
+}
+
 export async function setupBundlerAndFileWatcher(options: FileWatcherOptions) {
```

Comment:
Referencing [comment in original PR](https://github.com/Shopify/cli/pull/4812/files#r1836217274) - this function will go away. Where do we move this logic? @isaacroldan